CONTRIBUTING.md

```diff
@@ -17,10 +17,17 @@ Help us keep Angular open and inclusive. Please read and follow our [Code of Con
 
 ## <a name="question"></a> Got a Question or Problem?
 
-If you have questions about how to *use* Angular Material, please direct them to the 
-[Google Group][material-group] discussion list or [StackOverflow][stackoverflow]. 
-Please note that Angular Material is still in very early development, and the team's capacity 
-to answer usage questions is limited. Community chat is also available on [Gitter][gitter].
+Please, do not open issues for the general support questions as we want to keep GitHub issues for bug reports and feature requests. You've got much better chances of getting your question answered on [StackOverflow](https://stackoverflow.com/questions/tagged/angular-material2) where the questions should be tagged with tag `angular-material2`.
+
+StackOverflow is a much better place to ask questions since:
+
+- there are thousands of people willing to help on StackOverflow
+- questions and answers stay available for public viewing so your question / answer might help someone else
+- StackOverflow's voting system assures that the best answers are prominently visible.
+
+To save your and our time we will be systematically closing all the issues that are requests for general support and redirecting people to StackOverflow.
```

Comment:
`To save your and our time we will be ...` -> `To save your and our time, we will be...`